docs/get-started/installation.md

```diff
@@ -47,158 +47,275 @@ Alternatively, to deploy debug builds of your app directly from Windows to your
-```
+    ```dotnetcli
+    dotnet workload list
+    ```
+
+    On Windows and macOS, you should see the `maui` workload ID listed alongside the installed version. However, if you've installed it through the Visual Studio Installer on Windows the following workload IDs are listed:
+
+    ```
+    android
+    maui-windows
+    maccatalyst
+    ios
+    ```
+
+    On Linux, you should see the `maui-android` workload ID listed alongside the installed version.
+
+## Set up target platforms
+
+To build and debug a .NET MAUI app, you'll need to have a valid target platform relative to your development machine's operating system. The following table lists the supported target platforms on each operating system:
 
 | Your Operating System | Supported Target Platforms |
 |---|---|
 | Windows | Windows, Android |
 | macOS | Android, iOS, macOS |
 | Linux | Android |
 
-### iOS and macOS
+Building a .NET MAUI app for Android, and for Apple platforms, requires you to perform additional set up and configuration.
 
-To debug to an iOS or macOS target in Visual Studio Code:
+### Android
 
-1. Install the version of Xcode that's required by the version of .NET MAUI that you're using. For information, see [Release versions](https://github.com/dotnet/maui/wiki/Release-Versions). The latest stable Xcode release can be downloaded from the [Apple App Store](https://apps.apple.com/us/app/xcode/id497799835?mt=12).
-1. Run `xcode-select --install` in a terminal to acquire the Xcode command line tools.
-1. Open Xcode, and ensure you accept any license agreements.
+To process to set up your machine for .NET MAUI development on Android with Visual Studio Code is:
 
-### Android
+- Download and install the Java SDK or later. For more information, see [Download and install the Java JDK](#download-and-install-the-java-sdk).
+- Download and install the Android SDK. For more information, see [Download and install the Android SDK](#download-and-install-the-android-sdk).
+- Download and install an Android emulator. For more information, see [Download and install an Android emulator](#download-and-install-an-android-emulator).
 
-To debug to an Android target in Visual Studio Code:
+#### Download and install the Java SDK
 
-1. Install [Microsoft OpenJDK 17](/java/openjdk/download#openjdk-17).
-1. Install the Android SDK via one of the following approaches:
-    * **(Recommended)** Create a new .NET MAUI project (`dotnet new maui`) and use the [InstallAndroidDependencies target](#using-the-installandroiddependencies-target).
-    * Install via Visual Studio (Windows only).
-    * Install via Android Studio.
-    * Install via your preferred package manager on Linux.
+To download and install the Java SDK, and configure Visual Studio Code to use it:
 
-## Troubleshooting
+1. Download and install [Microsoft OpenJDK 17.0 or later](/java/openjdk/download). For information about installing the OpenJDK, see [Install the Microsoft Build of OpenJDK](/java/openjdk/install).
```

Comment:
"Microsoft OpenJDK 17.0 or later": Is this correct? I recently tried OpenJDK 21, and it didn't work.

Reply:
The .NET MAUI extension (In VSCode) delegates the Java SDK version to the Android workload, which is OpenJDK 17. The short version of the answer is that .NET for Android can't build with JDK 21 (yet).

There's a draft PR you can watch that's adding OpenJDK 21 support - https://github.com/dotnet/android/pull/9651

Reply:
Rather than installing OpenJDK manually, you could rely on the InstallAndroidDependencies build target to always install the correct JDK version.

Reply:
Since version 21 is not supported, a good documentation change would be from "Microsoft OpenJDK 17.0 or later" to "Microsoft OpenJDK 17".

Reply:
Already done.